Configuration.php

```diff
@@ -70,29 +70,61 @@ public function shouldThrowExceptionOnIpRangeSync()
      * @return array
      */
     public function getIpRangesAlwaysAllowed()
```

Comment:
Ooh, we can add typing to this function!  (nitpick)

```suggestion
    public function getIpRangesAlwaysAllowed(): array
```